samples/teacherNotebook.dib

```diff
@@ -12,7 +12,7 @@ double GetConstant() => 1.0;
 
 #!markdown
 
-## [Frontstage] (Stuff that comes AFTER THIS is what the student sees, everything else is BACKSTAGE, not student facing) (students wont see this line)
+## [Frontstage] (Stuff that comes after this is what the student sees, everything else is backstage, not student facing) (students wont see this line)
```

Comment:
```suggestion
## [Frontstage] (Stuff that comes after this is what the student sees, everything else is backstage, not student facing) (students won't see this line)
```

Reply:
Does this literally mean that the string `[Frontstage]` appears in the notebook and signifies that the following code is visible to the student?